app/views/server/unreachable.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="text-center mt-center">
+  <h2 class="text-muted"><%= I18n.t("system_out_of_air") %></h2>
+  <p class="text-muted"><%= I18n.t("working") %></p>
+</div>
+
+<div class="text text-center">
+  CONECTANDO
```

Comment:
Vale um i18n aqui? 

Sugestão: `I18n('attempt-to-connect')` puxando do arquivo server.pt-BR.yml